src/openrct2-ui/windows/EditorInventionsList.cpp

```diff
@@ -640,36 +635,18 @@ class InventionDragWindow final : public Window
     {
         auto screenCoords = windowPos + ScreenCoordsXY{ 0, 2 };
 
-        auto [drawString, ft] = WindowEditorInventionsListPrepareName(_draggedItem, true);
-        DrawTextBasic(&dpi, screenCoords, drawString, ft, { COLOUR_BLACK | COLOUR_FLAG_OUTLINE });
+        DrawResearchItem(
+            dpi, _draggedItem, width, screenCoords, STR_WINDOW_COLOUR_2_STRINGID, { COLOUR_BLACK | COLOUR_FLAG_OUTLINE });
     }
 
-    void Init(ResearchItem& researchItem)
+    void Init(ResearchItem& researchItem, const ScreenCoordsXY& editorPos, int scrollSize)
```

Comment:
I think `objectSelectionScrollWidth` or something like that is clearer than `scrollSize`.

Reply:
Agreed. Just updated.